jobrunner/secrets-template.env

```diff
@@ -0,0 +1,22 @@
+# These are secrets that should not be kept privately
```

Comment:
This is a comment which does not quite make sense ^

Should they be kept publicly? :)

Reply:
"should be kept private" was what I think I was going for